### ISSUE TITLE
BUG: A JavaScript error occured in the main process

### ISSUE THREAD

**Issue by dzavodnikov** [dbgate/dbgate#253]
**"A JavaScript error occured in the main process" popup window**

**To Reproduce**
1. Cretae connection to some DB.
2. Open tab with some query.
3. Wait some time.
4. Got popup window with exception (see "DbGate Error 1").
5. Press OK button.
6. Got popup window with exception (see "DbGate Error 2").
7. Press OK button and got popup window with exception (see screenshot 2) again and again.

**Expected behavior**
No exception on timout: automatically reconnect

**Screenshots**
![DbGate Error 1](https://user-images.githubusercontent.com/1470950/160303140-f98d866d-8687-46be-8a80-74c1cfc49f55.png)
![DbGate Error 2](https://user-images.githubusercontent.com/1470950/160303141-fcf42a5f-11eb-4ba5-aa1d-764b07b045a5.png)

**Version Information (please com plete the following information):**
 - OS: Windows 10
 - App Version: 4.7.4 (2022-03-25)
 - Install: Installer
 - Type: Application
 - Database engine: PostgreSQL

**Comment by janproch** [comment]
Thanks for error report, just question, do you use SSH tunnel? Or some other specific options?

I am not aware of any changes in 4.7.4 and 4.7.3 versions, which could possibly break this, I guess this problem is older.

**Comment by dzavodnikov** [comment]
Yes, my connection to DB in SSH tunnel. I am start using SSH in new version only, so myabe it is old bug.

**Comment by janproch** [comment]
OK, thanks to clarification. I will try to fix in this or next week.

**Comment by janproch** [comment]
Fixed in version 4.8.3. SSH forwarding was moved into separate subprocess, after unexpected error it should reconnect
